Sources/tart/VMStorageOCI.swift

```diff
@@ -16,10 +16,20 @@ class VMStorageOCI: PrunableStorage {
     VMDirectory(baseURL: vmURL(name)).initialized
   }
 
+  func digest(_ name: RemoteName) throws -> String {
+    let digest = vmURL(name).resolvingSymlinksInPath().lastPathComponent
+
+    if !digest.starts(with: "sha256:") {
+      throw RuntimeError.OCIStorageError("\(name) is not a digest and doesn't point to a digest")
+    }
```

Comment:
Does it mean that we'll be able to push only `sha256` digests and won't be able to push tags?

```bash
tart push ghcr.io/cirruslabs/macos-ventura-xcode:14.3-rc-2 ghcr.io/cirruslabs/macos-ventura-xcode:14.3
```

Reply:
The `resolvingSymlinksInPath()` above resolves any tag into a digest, so we'll be able to push both tags and digests.

It's just a sanity check to detect potential errors.